src/api/unified/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ target_link_libraries(af
     cpp_api_interface
     spdlog
     Threads::Threads
+    Boost::boost
```

Comment:
We don't use boost headers anywhere in unified API. If this it absolutely necessary for OSX build for some reason. It's better to have a comment around here so that we don't again remove it later.

Reply:
CHECK_ARRAYS uses AF_RETURN_ERROR which is defined in common/err_common.hpp which uses boost/stacktrace.hpp

Reply:
Clang caught it appropriately, hm.